src/main/java/edu/kis/powp/jobs2d/drivers/adapter/LineDrawerDriverAdapter.java

```diff
@@ -0,0 +1,39 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+/**
+ * driver adapter to drawer - special line.
+ */
+public class LineDrawerDriverAdapter implements Job2dDriver {
+    private int startX = 0, startY = 0;
+    private final DrawPanelController drawPanelController;
+
+    public LineDrawerDriverAdapter(DrawPanelController varDrawPanelController) {
+        drawPanelController = varDrawPanelController;
+    }
+
+    @Override
+    public void setPosition(int x, int y) {
+        this.startX = x;
+        this.startY = y;
+    }
+
+    @Override
+    public void operateTo(int x, int y) {
+        ILine line = LineFactory.getSpecialLine();
```

Comment:
Unfortunately, this is not a generic solution. (e.g. new line type requires new class, unnecessary direct coupling)